lib/elixir/lib/string.ex

```diff
@@ -240,37 +240,48 @@ defmodule String do
       iex> String.printable?("abc" <> <<0>>, 2)
       true
 
+      iex> String.printable?("abc" <> <<0>>, 0)
+      true
+
   """
-  @spec printable?(t) :: boolean
-  @spec printable?(t, non_neg_integer | :infinity) :: boolean
-  def printable?(string, counter \\ :infinity)
+  @spec printable?(t, 0) :: true
+  @spec printable?(t, pos_integer | :infinity) :: boolean
+
```

Comment:
Since there seems to be another round of changes coming up, this line shouldn't be here, we don't leave an empty line between specs and function. It's not added by the formatter I hope :D